<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViTables settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeView" name="pageSelector">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>161</width>
         <height>380</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="whatsThis">
        <string>Click an item in this tree to select the settings you want to change. Beware that the listed plugins are *loaded* plugins (i.e. the are working in the current ViTables session) not just enabled plugins.</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedPages">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="startupPage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis">
         <string>Configure the startup behavior of ViTables.</string>
        </property>
        <widget class="QGroupBox" name="startupGB">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>4</y>
           <width>351</width>
           <height>351</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="title">
          <string comment="The name of the groupbox where startup is configured">Startup</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="restoreCB">
            <property name="whatsThis">
             <string>If checked, the last working session -i.e. open files and views- will be restored. If not, ViTables will start with an empty session.</string>
            </property>
            <property name="text">
             <string comment="Label of the restore last session check button">Restore last session</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="lastDirCB">
            <property name="whatsThis">
             <string>The working directory used when opening the very first file. It defaults to the current directory if ViTables has been launched from the command line or to the user's home directory if ViTables started from a menu or a desktop icon. If the checkbox is checked then the last entered directory will be used.</string>
            </property>
            <property name="text">
             <string comment="Label of the startup in the last open directory check box">Start in last working directory</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="stylePage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis">
         <string>Configure the look and feel of ViTables.</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="loggerGB">
           <property name="whatsThis">
            <string>Customise font and colors of the Logger.</string>
           </property>
           <property name="title">
            <string comment="The title of the logger groubox">Logger</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1" rowspan="7">
             <widget class="QTextEdit" name="sampleTE">
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="whatsThis">
               <string>You can preview your Font settings here.</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table border=&quot;0&quot; style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;En un lugar de La Mancha,&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;de cuyo nombre no quiero acordarme,&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;no ha mucho tiempo vivia un hidalgo..&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::NoTextInteraction</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="foregroundPB">
              <property name="whatsThis">
               <string>Setup the text color.</string>
              </property>
              <property name="text">
               <string comment="The label of the logger text color button">F&amp;oreground</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="5" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="backgroundPB">
              <property name="whatsThis">
               <string>Setup the background color.</string>
              </property>
              <property name="text">
               <string comment="The label of the logger color button">&amp;Background</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="fontPB">
              <property name="whatsThis">
               <string>Here you can choose the font used in the Logger.</string>
              </property>
              <property name="text">
               <string comment="The label of the logger font chooser button">&amp;Font</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="workspaceGB">
           <property name="whatsThis">
            <string>Setup the background color of the workspace.</string>
           </property>
           <property name="title">
            <string comment="The title of the workspace groupbox">Workspace</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="workspacePB">
              <property name="whatsThis">
               <string/>
              </property>
              <property name="text">
               <string comment="The label of the workspace color button">&amp;Background</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="workspaceLabel">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="text">
               <string comment="A label for the workspace color sample">Background color sample</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="styleGB">
           <property name="whatsThis">
            <string>Choose the global style of ViTables here.</string>
           </property>
           <property name="title">
            <string comment="The title of the global aspect groupbox">Style</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QComboBox" name="stylesCB">
              <property name="whatsThis">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pluginsPage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QGroupBox" name="foldersGB">
           <property name="whatsThis">
            <string/>
           </property>
           <property name="title">
            <string>Available plugins</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QTreeView" name="pluginsTV">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="whatsThis">
               <string>The list of all available plugins. If a plugin is checked then it is enabled (but not necessarely loaded), if it is unchecked then it is disabled (but not necessarely unloaded). Please, note that changes in the check state of plugins will take effect the next time you start ViTables.</string>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <property name="expandsOnDoubleClick">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonsBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
